src/main/java/com/retrip/trip/application/in/response/ItineraryDetailsUpdateResponse.java

```diff
@@ -0,0 +1,28 @@
+package com.retrip.trip.application.in.response;
+
+import com.retrip.trip.domain.entity.Itinerary;
+import com.retrip.trip.domain.entity.ItineraryDetail;
+
+import java.util.List;
+import java.util.UUID;
+
+public record ItineraryDetailsUpdateResponse(
+        UUID itineraryId, List<ItineraryDetailUpdateResponse> itineraryDetails) {
+
+    public static ItineraryDetailsUpdateResponse of(Itinerary itinerary) {
+        return new ItineraryDetailsUpdateResponse(
+                itinerary.getId(),
+                itinerary.getItineraryDetails().getValues().stream()
+                        .map(
+                                id ->
+                                        new ItineraryDetailUpdateResponse(
+                                                id.getId(),
+                                                id.getLocationId(),
+                                                id.getPrice().getValue(),
+                                                id.getDescription().getValue()))
+                        .toList());
+    }
```

Comment:
포매팅이 기존 방식과 다른 부분이 있는것 같아요.
회의때 포매팅 방식 설정을 같이 해보면 좋겠네요.

Reply:
람다와 생성자를 같은 라인에 유지하거나 들여쓰기를 줄여서 중첩 깊이를 낮추는 쪽이 깔끔할것같습니다
```suggestion
public record ItineraryDetailsUpdateResponse(
        UUID itineraryId, List<ItineraryDetailUpdateResponse> itineraryDetails) {

    public static ItineraryDetailsUpdateResponse of(Itinerary itinerary) {
        return new ItineraryDetailsUpdateResponse(
                itinerary.getId(),
                itinerary.getItineraryDetails().getValues().stream()
                        .map(id ->
                                new ItineraryDetailUpdateResponse(
                                    id.getId(),
                                    id.getLocationId(),
                                    id.getPrice().getValue(),
                                    id.getDescription().getValue()))
                        .toList());
    }
```